### DISCUSSION TITLE
Teleport Upgrade version error: *trace.ConnectionProblemError

### DISCUSSION THREAD

**Discussion by nulldoot2k** [gravitational/teleport#49922] [Q&A] [answered]
> I using Ubuntu
> ```
> PRETTY_NAME="Ubuntu 22.04 LTS"
> NAME="Ubuntu"
> VERSION_ID="22.04"
> VERSION="22.04 LTS (Jammy Jellyfish)"
> VERSION_CODENAME=jammy
> ID=ubuntu
> ID_LIKE=debian
> HOME_URL="https://www.ubuntu.com/"
> SUPPORT_URL="https://help.ubuntu.com/"
> BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
> PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
> UBUNTU_CODENAME=jammy
> ```
> Meet same case
> when i upgrade version teleport from v16 to v17 then error but UI still work!
> ![Image](https://github.com/user-attachments/assets/85a45273-4b81-4c00-93a5-95124e7d0131)
> 
> What happen about `Original Error: *trace.ConnectionProblemError`
> and How can i fix it??? 

 _Originally posted by @nulldoot2k in [#9711](https://github.com/gravitational/teleport/issues/9711#issuecomment-2522303078)_

---
Expected behavior:

Current behavior:

Bug details:
- Teleport version
- Recreation steps
- Debug logs

**Comment by nulldoot2k** [comment, marked as answer]
@klizhentas Who do I contact to clarify this situation?

**Comment by zmb3** [comment]
@nulldoot2k you could start by including the appropriate troubleshooting information in the issue and we will do our best to try and help you.

If you can include debug logs and more information on your deployment and the steps you took to upgrade we might be able to provide suggestions.

**Comment by nulldoot2k** [comment]
@zmb3 Begin, i using version 16.0.4 for cluster.
I just upgrade 16.0.4 to v17
Step1: stop node auth2 and hold node auth1
when auth2 complete upgrade then auth1 keep.
Step2: when auth finish, then proxy upgrade same auth

After i see error up: *trace.ConnectionProblemError
This is construction i apply
![image](https://github.com/user-attachments/assets/3abc4f00-d77d-4120-86ef-7696085d76a8)
